### ISSUE TITLE
Missing Type Annotations

### ISSUE THREAD

**Issue by FoamyGuy** [adafruit/Adafruit_CircuitPython_RGB_Display#96]
There are missing type annotations for some functions in this library.

The `typing` module does not exist on CircuitPython devices so the import needs to be wrapped in try/except to catch the error for missing import. There is an example of how that is done here:

```python
try:
    from typing import List, Tuple
except ImportError:
    pass
```

Once imported the typing annotations for the argument type(s), and return type(s) can be added to the function signature. Here is an example of a function that has had this done already:

```python
def wrap_text_to_pixels(
    string: str, max_width: int, font=None, indent0: str = "", indent1: str = ""
) -> List[str]:
```
If you are new to Git or Github we have a guide about contributing to our projects here: https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github

There is also a guide that covers our CI utilities and how to run them locally to ensure they will pass in Github Actions here: https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code In particular the pages: `Sharing docs on ReadTheDocs` and `Check your code with pre-commit` contain the tools to install and commands to run locally to run the checks.

If you are attempting to resolve this issue and need help, you can post a comment on this issue and tag both @foamyguy and @kattni or reach out to us on Discord: https://adafru.it/discord in the `#circuitpython-dev` channel.

The following locations are reported by mypy to be missing type annotations:

- [ ] adafruit_rgb_display/rgb.py:42
- [ ] adafruit_rgb_display/rgb.py:53
- [ ] adafruit_rgb_display/rgb.py:72
- [ ] adafruit_rgb_display/rgb.py:75
- [ ] adafruit_rgb_display/rgb.py:83
- [ ] adafruit_rgb_display/rgb.py:91
- [ ] adafruit_rgb_display/rgb.py:99
- [ ] adafruit_rgb_display/rgb.py:120
- [ ] adafruit_rgb_display/rgb.py:134
- [ ] adafruit_rgb_display/rgb.py:150
- [ ] adafruit_rgb_display/rgb.py:154
- [ ] adafruit_rgb_display/rgb.py:158
- [ ] adafruit_rgb_display/rgb.py:162
- [ ] adafruit_rgb_display/rgb.py:171
- [ ] adafruit_rgb_display/rgb.py:203
- [ ] adafruit_rgb_display/rgb.py:221
- [ ] adafruit_rgb_display/rgb.py:225
- [ ] adafruit_rgb_display/rgb.py:229
- [ ] adafruit_rgb_display/rgb.py:239
- [ ] adafruit_rgb_display/rgb.py:249
- [ ] adafruit_rgb_display/rgb.py:288
- [ ] adafruit_rgb_display/rgb.py:299
- [ ] adafruit_rgb_display/st7789.py:99
- [ ] adafruit_rgb_display/st7735.py:124
- [ ] adafruit_rgb_display/st7735.py:186
- [ ] adafruit_rgb_display/st7735.py:271
- [ ] adafruit_rgb_display/ssd1351.py:100
- [ ] adafruit_rgb_display/ssd1331.py:108
- [ ] adafruit_rgb_display/ssd1331.py:136
- [ ] adafruit_rgb_display/s6d02a1.py:62
- [ ] adafruit_rgb_display/ili9341.py:79
- [ ] adafruit_rgb_display/ili9341.py:108
- [ ] adafruit_rgb_display/hx8357.py:95
- [ ] adafruit_rgb_display/hx8353.py:62

**Comment by ShootGan** [comment]
Is it still needed to do?

**Comment by kattni** [comment]
@ShootGan It sure looks like it! This is a big library, though! I simply wanted to give you a heads up on that, so you know what you're getting into. 

Please publish another post to this issue tagging both foamyguy (who created the issue above) and myself to let us know that you're picking it up. Thank you!

**Comment by matt-land** [comment]
Picking this up @FoamyGuy @kattni

**Comment by FoamyGuy** [comment]
closed by #111
